en/ui-system/components/editor/canvas.md

```diff
@@ -1,14 +1,14 @@
 # Canvas Component Reference
 
-The **Canvas** component is in charge of rendering all UI components, all UI elements must be placed under a __Canvas__ so that it can be rendered. The **Canvas** component can adapt the UI elements in __design resolution__ to the actual resolution of the device screen and can control zoom of all UI elements. There can be multiple __Canvas__ in the scene, each of them represents a camera, the rendering results of each canvas and 3d cameras can be presented in a certain order controlled by the `priority`.
+![](canvas/canvas.png)
 
-The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
+The node where the **Canvas** component is the root of the UI rendering. There can be multiple __Canvas__ in the scene. **all UI elements must be placed under a __Canvas__ so that it can be rendered**. Under which Canvas node, it is rendered by which Canvas. Each canvas can be presented in a certain order controlled by the `priority` attribute.
 
-![](canvas/canvas.png)
+In addition to the root node of the UI, it also has a screen adaptation function,  please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation. It also plays very important feature in the game production is the multi-resolution adaptation. The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
```

Comment:
In addition to the root node of the UI, it also has a screen adaptation function,  please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation. It also plays very important feature in the game production is the multi-resolution adaptation. The Canvas __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A camera is provided inside the __Canvas__, and the default z axis direction is __-1000 ~ 998__, so the z axis on the UI must be within this range to display properly (without taking the threshold value).
-->
In addition to the root node of the UI, __Canvas__ also has a screen adaptation function.  Please refer to the [Multi-Resolution Adaption](../engine/multi-resolution.md) documentation to read about this. __Canvas__ also supports multi-resolution adaptation. The __design resolution__ and __adaptation scheme__ are uniformly set in the __Project Setting__. A __Camera__ is provided inside the __Canvas__, and the default z-axis direction is __-1000 ~ 998__. The z-axis on the UI must be within this range to display properly (without taking the threshold value).